cmd/viewer/main.go

```diff
@@ -89,6 +91,9 @@ func run() error {
 	app.Get("/networks/:network/blocks", viewerHandler.ListBlocks)
 	app.Get("/networks/:network/blocks/:blockHash", viewerHandler.GetBlock)
 
+	app.Get("/transactions/networks/:network", viewerHandler.ListTransactions)
+	app.Get("/transactions/networks/:network/txHash/:txHash", viewerHandler.GetTransactionByTxHash)
+
```

Comment:
@Abirdcfly @bjwswang Any suggestions for this route?

Reply:
@Lanture1064 Route updates, please update documentation `doc/viewer_api.md`.

Reply:
> this

perhaps  
`/networks/:network/transactions/` 
`/networks/:network/transactions/:txHash`